docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -1,20 +1,20 @@
-# V2 Catalog Products and V3 Catalog APIs
+# Difference between V2 Products and V3 Catalog REST APIs
 
 <div class="otp" id="no-index">
 
 ### On this page
 - [V3 improvements](#v3-improvements)
-- [Difference between V2 Catalog Products and V3 Catalog APIs](#difference-between-v2-catalog-products-and-v3-catalog-apis)
+- [Difference between V2 Products and V3 Catalog APIs](#difference-between-v2-products-and-v3-catalog-apis)
 - [Related resources](#related-resources)
 
 </div>
 
-V2 Catalog Products and V3 Catalog REST APIs allow you to manage your store's physical and digital products.
+V2 and V3 Catalog REST APIs allow you to manage your store's products, categories, and brands, along with their sub-resources.
 
-Both the V2 Catalog Products and V3 Catalog APIs authenticate with OAuth and are designed to be used concurrently within a single application; however, data representation is significantly different.
-
-V2 and V3 REST APIs are not fully compatible. For differences in resources, see [Difference between V2 Catalog Products and V3 Catalog APIs](#difference-between-v2-catalog-products-and-v3-catalog-apis). When resources are available through both APIs, we recommend using the V3 API as it contains performance optimizations and usability improvements.
+Both the V2 Products and V3 Catalog APIs authenticate with OAuth and are designed to be used concurrently within a single application; however, data representation can be significantly different.
```

Comment:
Both the V2 Products and V3 Catalog APIs authenticate with OAuth and can be used concurrently within a single application; however, data representation can be significantly different.